projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -197,3 +226,14 @@ export function NewsletterAccess( { accessLevel, setPostMeta, withModal = true }
 		/>
 	);
 }
+
+export const NewsletterAccess = compose( [
+	withSelect( select => {
```

Comment:
As we are using hooks everywhere, I'd rather use `useSelect()` in the component instead of mixing paradigms....

Reply:
Yeah good point

Reply:
IMplemented